app/src/debug/res/layout/fragment_developer_settings.xml

```diff
@@ -157,6 +157,14 @@
 
         </LinearLayout>
 
+        <Button
```

Comment:
Can you please use same pattern with `LinearLayout`, styles and `TextView` on the left as other developer tools? For the button itself you can use style `?android:attr/borderlessButtonStyle` and write `show` as text, in `TextView` on the left you can write something like `LogCat`